.github/workflows/go_wasm_build_test_deploy.yml

```diff
@@ -9,29 +9,80 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  GO_VERSION: '1.22.5'
+  CHROME_VERSION: '129'
+  FIREFOX_VERSION: '131.0.3'
+  CHROME_DRIVER_VERSION: '129.0.6668.100'
+  GECKO_DRIVER_VERSION: '0.35.0'
+
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast
```

Comment:
Why are you opting not to fail fast?

Reply:
It would be easier to see if there any other failing tests in another OS. Leaving it to true, if one workflow fails, the rest will be cancelled

Reply:
Can you please add this as a comment to the file?